src/role-access/containers/exclusion/check-answers/check-answers.component.ts

```diff
@@ -1,31 +1,64 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { Store } from '@ngrx/store';
-import { ExclusionNavigationEvent, ExclusionState } from '../../../models';
+import { Component, Input, OnDestroy } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { Answer, ExcludeOption, ExclusionNavigationEvent, ExclusionState, ExclusionStateData } from '../../../models';
 import { ExclusionNavigation } from '../../../models/exclusion-navigation.interface';
 import * as fromFeature from '../../../store';
+import { AnswerHeaderText, AnswerLabelText } from '../../../models/enums';
 
 @Component({
   selector: 'exui-check-answers',
-  templateUrl: './check-answers.component.html',
-  styleUrls: ['./check-answers.component.scss']
+  templateUrl: './check-answers.component.html'
 })
-export class CheckAnswersComponent implements OnInit {
+export class CheckAnswersComponent implements OnDestroy {
 
   @Input() public navEvent: ExclusionNavigation;
 
+  public answers: Answer[] = [];
+  public caption = AnswerHeaderText.AddExclusion;
+  public heading = AnswerHeaderText.CheckAnswers;
+  public hint = AnswerHeaderText.CheckInformation;
+  public storeSubscription: Subscription;
+
+  private exclusionStateData: ExclusionStateData;
+
   constructor(private readonly store: Store<fromFeature.State>) {
+    this.storeSubscription = this.store.pipe(select(fromFeature.getRoleAccessState)).subscribe(exclusion => this.setAnswersFromExclusionStore(exclusion));
   }
 
-  public ngOnInit(): void {
+  public ngOnDestroy() {
```

Comment:
@paulhowes-HMCTS - just spotted a minor one here: return type should be `void`

Reply:
Added return type void

Reply:
Great - thanks, @paulhowes-HMCTS